.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
@wooneh is this file going to be used for something - or can we drop it?

Reply:
This is autogenerated by husky, so we should keep it.